mailbox/docker-image.nix

```diff
@@ -0,0 +1,30 @@
+{ dockerTools, python3, lib }:
```

Comment:
If you wanted this to use PyPy instead, you would use the name of the PyPy derivation here instead - eg "pypy3".

If you want to try this, I suggest using a very recent version of nixpkgs master@HEAD (or waiting for the 23.05 release).  I don't know if wormhole will have issues or not but I wasn't able to get Tahoe-LAFS to work on PyPy without a lot of hacking, even on the NixOS 22.11 release branch, due to a lot of Python package incompatibilities with PyPy that hadn't yet been addressed.  But after 22.11 it looks like someone has paid a bit closer attention to PyPy support and it's a little easier.

Reply:
Thank you for the tip.

If we want to compare Debian vs NixOS images, PyPy would help indeed.
But I'm not sure how much time it will take (time was also an important factor).